janitor/functions/select.py

```diff
@@ -327,17 +329,35 @@ def select_rows(
     return _select(df, rows=list(args), invert=invert)
 
 
+@register_groupby_method
+def pull(df: DataFrameGroupBy, label: Any) -> pd.Series:
```

Comment:
Is there a reason for this to be named `pull` instead of `select`? (I'd imagine with the latter, you'd probably have to move this out to a different `.py` file?)

Reply:
`groupby.select` returns a groupby object, while `groipby.pull` reaches further and returns a series. I am open to a better name (naming is hard 😁)

Reply:
I think we can park this for now (dont push this API) while we consider a better approach.  I've got some ideas that I'll probably push in a separate issue. I'll update the PR accordingly.

Reply:
@ericmjl I have excluded the `pull` function.